src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+export const checkStatus = (category) => ({
+  type: 'check Status',
+  payload: category,
+});
+
+const categoryReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'check status':
+      return 'Under construction';
```

Comment:
- [ ] Kindly add the action type and refactor your action creator in keeping with the standard. See example:

  ```
  const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
  const initialState = [];
  
  export const checkStatus = () => ({
    type: CHECK_STATUS,
  });
  
  const categoryReducer = (state = initialState, action) => {
    switch (action.type) {
      case CHECK_STATUS:
        return 'Under Construction';
  ...
  ```

Reply:
kindly plz Iam not understanding this section what exactly you want to see https://github.com/TedLivist

Reply:
Hi @Ibtisam34 I mean to be consistent
  - You should have an action type:
    ```
    const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'
    ```
  - And refactor your action creator (checkStatus) to use the action type you defined:
    ```
    export const checkStatus = () => ({
      type: CHECK_STATUS,
    });
    ```
  - Then refactor the reducer accordingly.